.ci/containers/membership-checker/reviewer_assignment.go

```diff
@@ -3,7 +3,10 @@ package main
 import (
 	"fmt"
 	"net/http"
+	"path/filepath"
+	"runtime"
 	"strings"
+	"text/template"
 
 	_ "embed"
```

Comment:
consider removing it since we are not using it anymore

Reply:
I'm switching back to embedding... more detail in the other thread.